src/main/java/io/pivotal/cla/mvc/admin/AdminClaController.java

```diff
@@ -43,12 +43,12 @@
 	@Autowired
 	protected ClaService claService;
 
-	protected List<ContributorLicenseAgreement> findPrimaryClas() {
+	protected List<ContributorLicenseAgreement> findPrimaryClass() {
```

Comment:
The plural of CLA (Contributor license agreement) is CLAs
In Java naming, that is Clas.